### DISCUSSION TITLE
Lines plugin: ignore repos

### DISCUSSION THREAD

**Discussion by Fuzuki785** [lowlighter/metrics#247] [Ideas]
Hello, this may not be the most useful idea on Earth, but it could still be nice, I think (I totally don't have a random million of lines committed because of a school project where I had to put our documents for some obscure reasons, I don't know what you mean).
There's probably some pertinence in having a global list of skipped repos too, rather than just for the languages plugin or individual plugins.

**Comment by lowlighter** [comment]
Hello,

Yes that's a nice idea, I added `plugin_lines_skipped` in #249 so you can ignore your school projects 😄 
There's also `repositories_skipped` option introduced in #251 which will populate all `plugin_*_skipped` options currently supported (languages and lines for now). If both are specified, repo lists will be merged together.

There's no equivalent to a global `plugin_*_skipped` because repositories are not always handled the same by each plugin. 
Some of them just cannot ignore some repositories because of the way some stats are retrieved so it would be kind of misleading so that's why I opted to a default list for individual option
